libs/sdk-ui-kit/styles/scss/messages.scss

```diff
@@ -0,0 +1,321 @@
+    cursor: pointer;
+
+    @include active-states {
+        color: inherit;
+        opacity: 1;
+    }
+}
+
+.gd-message-support {
+    @extend %gd-message-link;
+
+    margin-left: 10px;
+    text-decoration: underline;
+}
+
+.gd-message-dismiss {
+    @extend %gd-message-link;
+
+    position: relative;
+    top: 1px;
+    padding: 2px;
+    margin-left: 10px;
+
+    @include active-states-and-self {
+        text-decoration: none;
+    }
+}
+
+%gd-message-theme-black {
+    @include clearfix();
+
+    $message-success-borderColor: #6ba145;
+    $message-success-color: #2c6b00;
+    $message-success-bgcolor: #d6f6c8;
+
+    $message-error-borderColor: #d15462;
+    $message-error-color: #9c2d2d;
+    $message-error-bgcolor: #ffe7e7;
+
+    $message-warning-borderColor: #d18915;
+    $message-warning-color: #886407;
+    $message-warning-bgcolor: #fff9de;
+
+    box-sizing: border-box;
+    padding: 4px 6px;
+    border: 1px solid $message-error-borderColor;
+    font: 12px/1.3 $gd-font-primary;
+    color: $message-error-color;
+    background: $message-error-bgcolor;
+    pointer-events: all;
+    border-radius: 3px;
+    text-align: left;
+
+    &.error {
+        // error style is default
+    }
+
+    &.success {
+        color: $message-success-color;
+        background: $message-success-bgcolor;
+        border-color: $message-success-borderColor;
+    }
+
+    &.warning,
+    &.progress {
+        color: $message-warning-color;
+        border-color: $message-warning-borderColor;
+        background-color: $message-warning-bgcolor;
+    }
+
+    .gd-message-text {
+        color: #000;
+    }
+}
+
+%gd-message-overlay-theme-black {
+    @extend %gd-message-theme-black;
+
+    display: inline-block;
+
+    @include box-shadow("0 1px 2px 0 rgba(0, 0, 0, 0.15)");
+
+    .gd-message-support,
+    .gd-message-dismiss {
+        @extend %gd-message-link;
+
+        height: auto;
+        margin-left: 10px;
+        text-decoration: underline;
+
+        &::before {
+            content: "dismiss";
+            font-family: inherit;
+        }
+
+        @include active-states {
+            text-decoration: none;
+        }
+    }
+}
+
+/* ==========================================================================
+   Messages
+   ========================================================================== */
+
+.gd-message {
+    position: relative;
+    box-sizing: border-box;
+    display: inline-block;
+    padding: $gd-message-padding-vertical 8px $gd-message-padding-vertical 10px;
+    @include clearfix();
+
+    line-height: 1.4em;
+    color: $gd-color-negative;
+    background: transparentize($gd-color-negative, 0.85);
+    pointer-events: all;
+    max-width: 100%;
+    border-radius: 3px;
+    font-family: $gd-font-primary;
+    font-size: 14px;
+    text-align: left;
+
+    a:not(.gd-message-dismiss) {
+        color: inherit;
+        text-decoration: underline;
+    }
+
+    &.progress {
+        color: $gd-color-highlight;
```

Comment:
I would expect that this hardcoded primary color (all occurences) would be replaced by something more - customizable ;)

Reply:
Made customizable.